roles/ceilometer-common/tasks/monitoring.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: ceilometer-api check
```

Comment:
This process monitor, for ceilometer, should go into the ceilometer-control role.